### PR TITLE
improved responsiveness on boilerplate

### PR DESCRIPTION
**Pull request by diegolarraz**
## Objective
What: Responsiveness on Boilerplate
Why: It was broken
How: SCSS

## Type

- [x] Bug Fix
- [x] New Feature
- [x] Performance Tweaks
- [x] Style Tweaks
- [x] Code Refactoring

## Tested

Localhost

### CODE REVIEW

File: src/components/banners/SCSS/homeBanner.scss [hunk 1]
```diff
@@ -21,6 +21,7 @@
       height: 100%;
       margin: 1rem auto;
       object-fit: contain;
+      min-height: 170px;
```
Comment (VirAlex) [inline src/components/banners/SCSS/homeBanner.scss:24]: maybe just change the size of the h1 of the banner
Screenshot 2022-08-23 at 11 51 35
Screenshot 2022-08-23 at 11 51 53

**Review by VirAlex — COMMENTED** [review]
GG 👏

**Comment by VirAlex** [comment]
Really pretty and Smooth ;) 👏

**Review by matthew1809 — APPROVED** [review]
GG! Removed more code than you added!

**Pull request merged**